bot/client.js

```diff
@@ -57,4 +57,15 @@ client.on('message', (message) => {
   }
 });
 
+client.on('guildMemberAdd', (member) => {
```

Comment:
Narrowly scoped solution implemented. This is not reusable or extensible in the same ways that the ```client.on('message'``` is. There you can see that multiple controllers are ran against the ```message``` event to allow multiple processings. Currently, If another thing were to need to be implemented on the guildmemberadd event your proposed change would need to be rewritten heavily.